packages/textkit/src/layout/wrapWords.js

```diff
@@ -28,6 +28,43 @@ const hyphenateWordWrapper = (hyphenator, word) => {
   return hyphenator;
 };
 
+/**
+ * @summary TO handle the already hyphenated parts,
+                - This will break the parts which have hyphens into smaller parts
+ * @param parts- parts array from `hyphen`
+ *                
+ */
+const handleHyphenedWords = (parts)=>{
+     const newParts = [];
+
+     // eslint-disable-next-line no-plusplus
+     for (let curP=0 ; curP < parts.length;curP++) {
```

Comment:
```suggestion
     for (let curP=0; curP < parts.length; curP+=1) {
```

remove the disable comment please 

and check the indentation in this file (I think this line is indented by 5 spaces) and reformat